src/routes/moderation.rs

```diff
@@ -51,3 +52,103 @@ pub async fn get_projects(
 
     Ok(HttpResponse::Ok().json(projects))
 }
+
```

Comment:
If we provide a way for modders to put custom webhooks for releases, wouldn't it be better to move this to a utility file ?
This may also help to reduce the length of this file.